plot.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# `make` needs to be run twice, 
+# first to generate traineddata and then to generate new validate.log files.
+# $1 - MODEL_NAME
+# $2 - VALIDATE_LIST
+# $3 - maximum CER for y axis - adjust based on graph
+
+cd plot
+make MODEL_NAME=$1 VALIDATE_LIST=$2 Y_MAX_CER=$3
+make MODEL_NAME=$1 VALIDATE_LIST=$2 Y_MAX_CER=$3
```

Comment:
If you need a shell script to run a makefile, then that makefile is poorly written (or documented).

Assuming it is a good choice to have a separate makefile for plotting in the first place, I think the plot/Makefile should just require/assume that the toplevel Makfile's `traineddata` target has already been run.

In the very least, this should read `make traineddata && make plotvalidatecer`, not `make all; make all`.